resources/views/layout.blade.php

```diff
@@ -94,7 +94,7 @@
                     </li>
                 </ul>
 
-                @if(Auth::check() || Session::has('auth.vatsim-sso'))
+                @if(Auth::check() && Session::has('auth.vatsim-sso'))
```

Comment:
This should be `||` instead of `&&` - the logout route clears the session, so if they're partially authenticated (`auth.vatsim-sso`) but not actually logged in yet (`Auth::check()`), they can still log out and reset the session.